### DISCUSSION TITLE
Task translate

### DISCUSSION THREAD

**Discussion by wildwind1973** [Purfview/whisper-standalone-win#88] [Q&A] [answered]
From --help:
`whether to perform X->X speech recognition ('transcribe') or X->English translation ('translate') (default: transcribe)`
how can I translate from any language to any language from the list?
for example in German?

**Comment by Purfview** [comment, marked as answer]
Model is trained to do only translation to English.

**Comment by despairTK** [comment]
In fact, other languages can be translated. For example, your audio is English, but you want to output Japanese directly. You can set --language=Japanese directly, so the result of the output is Japanese text.

However, there are disadvantages to do this. One is that Whisper's language translation function itself is not accurate. The other is the translated subtitle text, and the timestamp will have a lot of dislocation. If your audio time is not very long, and the speaker in the audio is obviously reasonable. Then you can try the method I said.
